chai-as-promised/.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
There are no other .editorconfig files in the definitely typed repo. Probably best to exclude.